resource_obmcs_core_route_table.go

```diff
@@ -191,9 +201,14 @@ func (s *RouteTableResourceCrud) ExtraWaitPostCreateDelete() time.Duration {
 	return time.Duration(15 * time.Second)
 }
 
-func (s *RouteTableResourceCrud) buildRouteRules() (routeRules []baremetal.RouteRule) {
+func (s *RouteTableResourceCrud) buildRouteRules() (routeRules []baremetal.RouteRule, e error) {
 	routeRules = []baremetal.RouteRule{}
 	for _, val := range s.D.Get("route_rules").([]interface{}) {
+
+		if val == nil {
+			return nil, fmt.Errorf("Empty route_rules are not permitted, omit empty blocks entirely.")
```

Comment:
How about: "Empty route_rules are not permitted. Instead, the route_rules block may be omitted entirely."